scripts/utils/migrate/get-frontmatter.js

```diff
@@ -47,10 +47,13 @@ const addCustomFrontmatter = {
     ...defaultFrontmatter,
     topics,
   }),
-  [TYPES.API_DOC]: ({ topics }, defaultFrontmatter) => ({
-    ...defaultFrontmatter,
-    topics,
-  }),
+  [TYPES.API_DOC]: ({ doc, topics }, defaultFrontmatter) => {
```

Comment:
We probably don't need a `return` here.

Reply:
Mostly a demo of commenting within VS Code, but I suppose this is still a valid (allbeit nitpicky) comment.